### PR TITLE
Fix #6713 One-click-importer permission issue

### LINKED ISSUE #6713: One-click-importer permission issue for non-superusers
### How to Reproduce
Create a user with:
- View permissions on one click importer plugin, OneClickImporterJobExecution
- View permissions on dataexplorer and dataexplorer-settings and genomebrowsersettings and attributes
- Write permissions on attribute, entityType, Package
- Login as user
- Import a simple csv or xlsx
- Click the link to the dataexplorer

### Expected behavior
I can see my data

### Observed behavior
Error
After looking at permissions on the EntityType, I can see the user has none

### PR DESCRIPTION
**Pull request by dennishendriksen**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated
- [x] Clean commits

### CODE REVIEW

File: molgenis-security/src/main/java/org/molgenis/security/permission/PermissionSystemServiceImpl.java [hunk 1]
```diff
@@ -61,15 +64,36 @@ public void giveUserWriteMetaPermissions(Collection<EntityType> entityTypes)
...
+
+	private void updateSecurityContextAuthorities(SecurityContext securityContext,
+			Collection<? extends GrantedAuthority> newAuthorities)
+	{
+		Collection<? extends GrantedAuthority> authorities = securityContext.getAuthentication().getAuthorities();
+		List<GrantedAuthority> updatedAuthorities = new ArrayList<>(authorities);
+		updatedAuthorities.addAll(newAuthorities);
+
+		Authentication authentication = securityContext.getAuthentication();
+		UsernamePasswordAuthenticationToken newToken = new UsernamePasswordAuthenticationToken(
```
Comment (fdlk) [inline molgenis-security/src/main/java/org/molgenis/security/permission/PermissionSystemServiceImpl.java]: I think that means that they'll need to re-enter the 2FA code if 2FA is enabled.

**Comment by fdlk** [comment]
Functional question: Do we want all of the user's sessions to be updated, or only the one that triggered the import job?

File: molgenis-security/src/main/java/org/molgenis/security/permission/PermissionSystemServiceImpl.java (hunk 1 above)
Comment (dennishendriksen) [reply molgenis-security/src/main/java/org/molgenis/security/permission/PermissionSystemServiceImpl.java]: Oops, I wasn't aware of the fact that tokens could be of a different type. This fix indeed doesn't work for 2FA.

**Comment by sidohaakma** [comment]
In the PermissionSystemServiceImpl use requireNonNull on this line:
this.authenticationAuthoritiesUpdater = authenticationAuthoritiesUpdater;

**Pull request merged**